icefoganalytics/travel-authorization
# Add Entity Relationship Diagram in PlantUML

# Context

Better illuminate the state of data management in this project.

# Implementation

Add entity relationship diagrams in plantuml.

# Screenshots

![Entity Relationship Diagram](https://github.com/icefoganalytics/travel-authorization/assets/23045206/7520630f-773d-43f1-a522-1c210a2fb6ed)

# Testing Instructions

1. Boot the app with `dev up`
2. Open the file `_Design/Entity Relationship Diagram.wsd` in VS Code.
3. Check that the `PlantUML: Preview Current Diagram` command loads the preview.
4. Make some changes and check that the preview refreshes.
5. Check that you can export the diagram.
